src/Streamly/Internal/Data/Stream/IsStream/Common.hs

```diff
@@ -547,12 +547,12 @@ concatM generator = concatMapM (\() -> generator) (fromPure ())
 --
 -- 'splitOnSeq' is an inverse of 'intercalate'. The following law always holds:
 --
--- > intercalate . splitOn == id
+-- prop> intercalate . splitOn == id
 --
 -- The following law holds when the separator is non-empty and contains none of
 -- the elements present in the input lists:
 --
--- > splitOn . intercalate == id
+-- prop> splitOn . intercalate == id
```

Comment:
The description above says this does not always hold, so this cannot be marked with `prop>`.